content/docs/start/experiments/experiment-versioning.md

```diff
@@ -0,0 +1,224 @@
+---
+title: 'Get Started: Experiment Versioning'
+description:
+  'Get started with versioning experiments with DVC. Track the changes to the
+  code, data, metrics, parameters and plots associated with each experiment,
+  without bloating your Git repo.'
+---
+
+# Get Started: Experiment Versioning
+
+The early stages of a Machine Learning (ML) project are usually exploratory and
+demand fast iterations. Many ML practitioners chose to use tools that require
+little setup and enable rapid prototyping, like those based on
+[Jupyter Notebooks](https://jupyter.org/).
+
+Although these tools are extremely useful, keeping track of the changes you made
+across experiments quickly becomes cumbersome and reproducing an experiment can
+be a challenge.
+
+You can start using DVC to version your experiments without leaving your Jupyter
+Notebook. There are no logins, servers, databases, or UI to spin up. Every
+experiment will be versioned **without cluttering your repo**, unlike saving
+each run to a separate directory or creating a Git branch for each.
+
+## Enable Versioning
+
+All you need to start is a <abbr>DVC repository</abbr> and the [DVCLive] Python
+library installed:
+
+```cli
+$ pip install dvclive
+```
+
+In your Python code, you can start versioning your experiments with
+`save_dvc_exp=True`.
```

Comment:
```suggestion
In your Python code, you can start versioning your experiments in
DVCLive's `Live` API or framework-specific callbacks with `save_dvc_exp=True`.
```

Minor, not blocking suggestion to stress the overall API here since this is not a DVCLive-specific page and that context may not be so obvious.